xbmc/cores/VideoPlayer/VideoPlayerRadioRDS.cpp

```diff
@@ -819,7 +819,6 @@ void CDVDRadioRDSData::ProcessUECP(const unsigned char *data, unsigned int len)
               case UECP_RDS_PS:                 ret = DecodePS(msg);                  break;  //!< Program Service name (PS)
               case UECP_RDS_DI:                 ret = DecodeDI(msg);                  break;  //!< Decoder Identification and dynamic PTY indicator
               case UECP_RDS_TA_TP:              ret = DecodeTA_TP(msg);               break;  //!< Traffic Announcement and Traffic Programme bits.
-              case UECP_RDS_MS:                 ret = DecodeMS(msg);                  break;  //!< Music/Speech switch
```

Comment:
If unused change it to:
´´´cpp
               case UECP_LINKAGE_INFO:           ret = 4;                              break;   //!< Music/Speech switch
´´´
To prevent stream flow pointer failure!